src/lib/usbstick.ts

```diff
@@ -0,0 +1,50 @@
+export const isAvailable = () => {
+  return !!window.kiosk
+}
+
+export const isPresent = async () => {
+  return isAvailable() && (await window.kiosk!.getUsbDrives()).length > 0
+}
+
+const getDevice = async () => {
+  return (await window.kiosk!.getUsbDrives())[0]
+}
+
+const isDeviceMounted = (device: KioskBrowser.UsbDrive) => {
```

Comment:
If `device` might not be set, do `device?: …`. You can also do `Boolean(device?.mountPoint)`.